app/src/main/java/org/p2p/wallet/settings/ui/recovery/RecoveryKitPresenter.kt

```diff
@@ -34,4 +58,26 @@ class RecoveryKitPresenter(
             showSocialId(notAvailableString)
         }
     }
+
+    private fun fetchMetadata(seedPhraseProviderType: SeedPhraseProviderType?) {
```

Comment:
is not used except for check in the beginning
you can extract the check to the call site